pkg/app/context.go

```diff
@@ -1045,6 +1045,20 @@ func (ctx *RequestContext) Cookie(key string) []byte {
 	return ctx.Request.Header.Cookie(key)
 }
 
+// Cookies returns all the request cookies.
+//
+// It's a good idea to call protocol.ReleaseCookie to reduce GC load after the cookie used.
+func (ctx *RequestContext) Cookies() []*protocol.Cookie {
```

Comment:
Is it necessary add this method in `RequestContext`? Not all methods should be add in `RequestContext`

Reply:
remove to header.go, thx